wiki/Kiai_time/id.md

```diff
@@ -0,0 +1,21 @@
+stub: true
+tags:
```

Comment:
The header here is missing the "---", apply this one:
```
---
stub: true
tags:
  - timing section
  - kiai mode
  - waktu
---

Reply:
ok, applied